### PR TITLE
Feat/cse 370 add example

### PR DESCRIPTION
**Pull request by tnoelcke**
# Changes introduced

This pr introduces the concept of an example in the task models. This includes code to parse fractions through out the target as well as code to parse in images from the examples.

## Related issue(s):

- CSE-370

## Contributor checklist

- [x] Wrote/updated tests for introduced changes
- [x] Added new stories for created/modified components (if applicable)
- [ ] Checked Storybook for Accessibility issues (if applicable)
- [ ] ~~Updated Postman library for created/modified routes (if applicable)~~
- [x] Verified that there are no issues in CircleCI
- [x] Assigned yourself to the PR
- [x] Removed `WIP:` from the PR title
- [x] Requested review from the SB Reviewers team

## Reviewer checklist

- [ ] Assigned yourself to the PR
- [ ] Read through the changes in the `Files changed` tab
- [ ] Verified that tests were written/modified
- [ ] Verified that stories were written/modified (if applicable)
- [ ] ~~Verified that Postman was updated (if applicable)~~
- [ ] Checked out the branch and tested changes locally
- [ ] Run storybook and checked that there are no accessibility issues

### CODE REVIEW

File: src/components/TargetComponents/TargetDetail.tsx [hunk 1]
```diff
@@ -32,6 +33,7 @@ function stemSection(stem: IStem[]): ContentSection[] {
 
   return sections;
 }
+
 // tslint:disable: no-non-null-assertion
 function handleFractions(taskModels: ITaskModel[]) {
```
Comment (danvanhorn) [inline src/components/TargetComponents/TargetDetail.tsx]: I was going to resume development on this but I'm not sure about the scope of the task and there isn't a task for this in Jira. Should we remove this method or does it have to be included elsewhere?

**Comment by danvanhorn** [comment]
I rebased this PR against `dev` to resolve conflicts. You'll probably want to check that I didn't undo anything.

File: src/components/TargetComponents/TargetDetail.tsx (hunk 1 above)
Comment (tnoelcke) [reply src/components/TargetComponents/TargetDetail.tsx]: I agree, this is really a parser and belongs in the main content parse util file. So I moved it there and added it to the parser list.

Comment (puregarlic) [reply src/components/TargetComponents/TargetDetail.tsx]: Since you replaced this function, it needs to be removed.

**Review by puregarlic — APPROVED** [review]
Alright, good work. Thanks for utilizing the parser pipeline.

**Pull request merged**